chainer/utils/sparse.py

```diff
@@ -0,0 +1,72 @@
+import chainer
+from chainer.backends import cuda
+
+
+class CooMatrix(object):
```

Comment:
Could you add documentations to this class, `to_dense` and `to_coo`?

Reply:
All right, will add documentations to `CooMatrix`, `to_dense` and `to_coo`.